tus_storagehandler/config.yaml

```diff
@@ -0,0 +1,40 @@
+server:
+    host: '0.0.0.0'
+    port: 8080
+    debug: False
+    environment: development
+    testing: False
+    use_reloader: True
+
+security:
+    auth:
+        required: True
+        add_key_to_claims: True
+        allow_expired: False
+        audience: null
+        claim_identity: sub
+        claim_issuer: iss
+        algorithms:
+          - RS256
+        validation_methods:
+          - userinfo
+          - public_key
+        validation_checks: all
+
+api:
+    specs:
+        - path: storage_handler.yaml
+          append: null
+          add_operation_fields:
+              x-openapi-router-controller: controllers
+          connexion:
+              strict_validation: True
+              validate_responses: False
+              options:
+                  swagger_ui: True
+                  serve_spec: True
+
+exceptions:
+    required_members: [['message'], ['code']]
+    status_member: ['code']
+    exceptions: exceptions.exceptions
```

Comment:
You will get an error if you start up the service if `exceptions.exceptions` is not available, so please add it here (not in a separate PR).